templates/zerver/footer.html

```diff
@@ -1,186 +1,108 @@
+        <div class="footer__section">
+            <span class="footer__section-title">
+                {{ _("Solutions") }}
+            </span>
+            <p>
                 <a href="/for/business/">{{ _("Business") }}</a>
-            </li>
-            <li>
                 <a href="/for/education/">{{ _("Education") }}</a>
-            </li>
-            <li>
                 <a href="/for/research/">{{ _("Research") }}</a>
-            </li>
-            <li>
-                <a href="/for/events/">{{ _("Events and conferences") }}</a>
-            </li>
-            <li>
+                <a href="/for/events/">{{ _("Events & conferences") }}</a>
                 <a href="/for/open-source/">{{ _("Open source projects") }}</a>
-            </li>
-            <li>
                 <a href="/for/communities/">{{ _("Communities") }}</a>
-            </li>
-        </ul>
-    </div>
-    <div class="footer-section">
-        <h3>{{ _("Customer stories") }}</h3>
-        <ul>
-            <li>
-                <a href="/case-studies/idrift/">iDrift AS {{ _("Company") }}</a>
-            </li>
-            <li>
-                <a href="/case-studies/tum/">{{ _("Technical University of Munich") }}</a>
-            </li>
-            <li>
-                <a href="/case-studies/ucsd/">{{ _("University of California San Diego") }}</a>
-            </li>
-            <li>
-                <a href="/case-studies/lean/">{{ _("Lean theorem prover community") }}</a>
-            </li>
-            <li>
-                <a href="/case-studies/asciidoctor/">{{ _("Asciidoctor open-source community") }}</a>
-            </li>
-            <li>
-                <a href="/case-studies/rust/">{{ _("Rust language community") }}</a>
-            </li>
-            <li>
-                <a href="/case-studies/recurse-center/">Recurse Center</a>
-            </li>
-            <li>
-                <a href="/communities/">{{ _("Open communities directory") }}</a>
-            </li>
-        </ul>
-    </div>
-    <div class="footer-section">
-        <h3>{{ _("Accounts") }}</h3>
-        <ul>
-            {% if find_team_link_disabled %}
-            {% else %}
-            <li><a href="{{ root_domain_uri }}/accounts/find">{{ _("Find accounts") }}</a></li>
-            {% endif %}
-            {% if open_realm_creation %}
-            <li><a href="{{ root_domain_uri }}/new/">{{ _("New organization") }}</a></li>
-            {% endif %}
-            <li><a href="{{ root_domain_uri }}/terms/">{{ _("Terms of Service") }}</a></li>
-            <li><a href="{{ root_domain_uri }}/privacy/">{{ _("Privacy policy") }}</a></li>
-        </ul>
-    </div>
-    <div class="footer-section">
-        <h3>{{ _("For users") }}</h3>
-        <ul>
-            <li>
+            </p>
+            <p>
+                <a href="https://zulip.com/use-cases/">{{ _("Customer stories") }}</a>
+                <a href="https://zulip.com/communities/">{{ _("Open communities") }}</a>
```

Comment:
These links look to have been accidentally converted from relative to hardcoding zulip.com.

Reply:
Yeah, for some reason I intentionally converted them but not sure why.